src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -68,11 +68,14 @@ public class EditCommand extends Command {
     private final Index index;
     private final EditPersonDescriptor editPersonDescriptor;
 
+    private final PersonType personType;
+
     /**
-     * @param index of the person in the filtered person list to edit
+     * @param index                of the person in the filtered person list to edit
      * @param editPersonDescriptor details to edit the person with
+     * @param personType The type of person being edited i.e. patient or specialist
      */
-    public EditCommand(Index index, EditPersonDescriptor editPersonDescriptor) {
+    public EditCommand(Index index, EditPersonDescriptor editPersonDescriptor, PersonType personType) {
```

Comment:
Hm, how come we are adding `PersonType` as a parameter to the `EditCommand` here, I thought we were ignoring the `PersonType`? Since Edit is not a priority for now, we could also leave it as it is and create an issue for it so we know to resolve it in the future.

Reply:
You're right, will make the changes!